### PR TITLE
[NEW] add `javascript/happyx` (NodeJS)

### PR DESCRIPTION
**Pull request by Ethosa**
This will works only on Linux.

I tested it on Ubuntu `22.04`

```shell
npm install happyx
npm i
node app.js
```

### CODE REVIEW

File: javascript/happyx/config.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  version: 1.1.1
+
+  bootstrap:
+    - npm install happyx
+    - npm install
+
+  command: node app.js
+
+  engines:
+    - node
```
Comment (waghanza) [inline javascript/happyx/config.yaml:13]: no need `node` is the default engine here

File: javascript/happyx/config.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+framework:
+  github: Hapticx/happyx
+  website: hapticx.github.io/happyx
+  version: 1.1.1
+
+  bootstrap:
+    - npm install happyx
+    - npm install
+
+  command: node app.js
```
Comment (waghanza) [inline javascript/happyx/config.yaml:10]: we use pm2 for node based, you might remove it

File: javascript/happyx/config.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+framework:
+  github: Hapticx/happyx
+  website: hapticx.github.io/happyx
+  version: 1.1.1
+
+  bootstrap:
+    - npm install happyx
+    - npm install
```
Comment (waghanza) [inline javascript/happyx/config.yaml:8]: no need, this is kind a standard and will be done (there is twice this command in generated dockerfile)
see `bundle exec rake config`

File: javascript/happyx/config.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+framework:
+  github: Hapticx/happyx
+  website: hapticx.github.io/happyx
+  version: 1.1.1
```
Comment (waghanza) [inline javascript/happyx/config.yaml:4]:
```suggestion
  version: 1.1
```

File: javascript/happyx/app.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  return req.params.id;
+});
+
+app.post("/user", (req) => {
+  return "";
+});
+
+
+// start application
+app.start();
```
Comment (waghanza) [inline javascript/happyx/app.js:22]: do you listen on `0.0.0.0:3000` ?

**Comment by waghanza** [comment]
Seems that there is no timeout (an open connection but no return) onto the container

**Comment by Ethosa** [comment]
@waghanza have you some idea how to fix it? 🤔

**Comment by waghanza** [comment]
Will try something later today

**Pull request closed without merging**